src/ToggleSwitch.scss

```diff
@@ -26,7 +26,6 @@
     background: #F1F1F1;
     border-radius: 50%;
     display: inline-block;
-    margin-bottom: 0.1rem;
```

Comment:
Apparently this margin has no effect at all -- I changed it to 100rem and still look the same, which means that we can get rid of margin-top on next line as well.

Reply:
I cleaned this up and a similar one in the Audio Feedback toggle.  I checked the expand add node as well.